hikari/internal/ux.py

```diff
@@ -233,6 +240,16 @@ def print_banner(
         stdout.write(string.Template(raw_banner).safe_substitute(args))
 
 
+def warn_if_not_optimized() -> None:
+    """Log a warning if not running in optimization mode."""
+    if __debug__:
+        _LOGGER.warning(
+            "You are running on no-optimization level, which may speed-down your application. "
```

Comment:
```suggestion
            "You are not running on any optimization level, which may slow down your application. "
```

Reply:
Would this be better?

```py
        "You are running on optimization level 0 (no optimizations), which may slow down your application. "
```

as it's not that you are not running on an optimization level, you are just running on optimization level 0.

Reply:
I think that works!

Reply:
Cool, I'll change it when I have a computer to ensure it doesn't go over the character limit.

Thanks for the suggestion!